src/briefcase/macos.py

```diff
@@ -58,6 +62,14 @@ def launcher_script_location(self):
         return macos_dir
 
     def build_app(self):
+        print("Building DMG file...")
+        dmg_name = self.formal_name + '.dmg'
+        dmg_path = os.path.join(os.path.abspath(self.dir), dmg_name)
+        settings = {'files': [self.app_location],
+                    'symlinks': {'Applications': '/Applications'}}
+        dmgbuild.build_dmg(filename=dmg_path,
```

Comment:
At column 1: (W293) blank line contains whitespace

Reply:
What blank line? @freakboy3742 could you look at this?

Reply:
Huh?